src/dispatcher.js

```diff
@@ -299,10 +306,19 @@ var dispatcher = {
     return eventCopy;
   },
   getTarget: function(inEvent) {
-
-    // if pointer capture is set, route all events for the specified pointerId
-    // to the capture target
-    return this.captureInfo[inEvent.pointerId] || inEvent._target;
+    var capture = this.captureInfo[inEvent.pointerId];
+    if (!capture) {
+      return inEvent._target;
+    }
+    if (!this.suppressDuringCapture(inEvent)) {
+      return capture;
+    }
+  },
+  suppressDuringCapture: function(inEvent) {
+    var capture = this.captureInfo[inEvent.pointerId];
+    var isNonCaptureTarget = inEvent._target !== capture;
+    var needsSuppressing = BOUNDARY_EVENTS.indexOf(inEvent.type) > -1;
```

Comment:
Turned `BOUNDARY_EVENTS` into an object: https://github.com/jquery/PEP/blob/89149e0595203a3425686f93100fbf3f3e759d1c/src/dispatcher.js#L88-L93 . Does it need a `Object.create(null)`, for defensive reasons?